Qt/InstaDam/Selector/freeDrawSelect.h

```diff
@@ -34,6 +34,7 @@ class FreeDrawSelect : public QGraphicsPixmapItem, public SelectItem {
         return FreeDrawSelect::baseInstruction;
     }
     QRectF boundingRect() const override;
+    void setOpacity(qreal val);
```

Comment:
5: Cannot find the User Story that the change implements.

Reply:
Please see the revised user story list here (User Story 37)